# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/ipc/GPUParent.cpp
# Commit: ec0e9a72aebb
# Full Hash: ec0e9a72aebb72422b06c1cadee2b3f7dc3538ce
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2020-01-10 05:03:08
# Regressor Bug: 1600472
# File Overlap Count: 2
# Description:
#   Bug 1600472 - Disable allowing sacrificing subpixel anti-aliasing for small screens. r=jrmuizel
#   
#   This patch only allows sacrificing subpixel anti-aliasing when the
#   screen size is larger than WUXGA, and when the force disable pref is not
#   set. In the future, we may also add disable this for high end GPUs.
# ==============================================================================

diff -r 912fed7c5710 -r ec0e9a72aebb gfx/ipc/GPUParent.cpp
--- a/gfx/ipc/GPUParent.cpp	Thu Jan 09 14:14:08 2020 +0000
+++ b/gfx/ipc/GPUParent.cpp	Thu Jan 09 15:23:38 2020 +0000
@@ -134,6 +134,7 @@
   APZThreadUtils::SetControllerThread(MessageLoop::current());
   apz::InitializeGlobalState();
   LayerTreeOwnerTracker::Initialize();
+  CompositorBridgeParent::InitializeStatics();
   mozilla::ipc::SetThisProcessName("GPU Process");
 #ifdef XP_WIN
   wmf::MFStartup();